.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ develop ]
+  push:
+    branches: [ develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '8.0.x'
+        
+    - name: Restore dependencies
+      run: dotnet restore
+      
+    - name: Build
+      run: dotnet build --no-restore
+      
+    - name: Test
+      run: dotnet test --no-build --filter "Category!=Osm.Integration" --verbosity normal
```

Comment:
Do not skip any tests in the pipeline definition. If any tests need to be skipped, use `Skip` property on the `Fact` attribute and add reason, why the test is skipped.